File: App Examples/ObjCApp/ObjCApp/ViewController.m
//
//  ViewController.m
//  ObjCApp
//
//  Created by Wojciech Mandrysz on 15/06/16.
//  Copyright © 2016 Evokly. All rights reserved.
//

#import "ViewController.h"
@import Evokly;

@implementation ViewController

- (IBAction)showDebug {
    [Evokly presentDebugViewControllerWithParent:self];
}

@end

File: App Examples/SwiftApp/SwiftApp/ViewController.swift
//
//  ViewController.swift
//  SwiftApp
//
//  Created by Wojciech Mandrysz on 15/06/16.
//  Copyright © 2016 Evokly. All rights reserved.
//

import UIKit
import Evokly

class ViewController: UIViewController {

    @IBAction func showDebug() {
        Evokly.presentDebugViewController(parent: self)
    }

}
